templates/deb/changelog.erb

```diff
@@ -2,4 +2,4 @@
 
   * Package created with FPM.
 
- -- <%= maintainer %>  <%= Time.now.strftime("%a, %d %b %Y %T %z") %>
+ -- <%= maintainer %>  <%= (if attributes[:source_date_epoch].nil? then Time.now() else Time.at(attributes[:source_date_epoch].to_i) end).strftime("%a, %d %b %Y %T %z") %>
```

Comment:
%a ceased being locale-specific in ruby 1.9....? 
https://idiosyncratic-ruby.com/57-what-the-time.html says
```
"%a | Weekday Name (Abbreviated)
Locale-independent (English) three-letter name of weekday"
```

Verified:
```
$ export DEBEMAIL=dank@kegel.com
$ export DEBFULLNAME="Dan Kegel"
$ export SOURCE_DATE_EPOCH=123
$ LANG=de_DE.UTF-8
$ date
Do 22. Jun 15:42:00 PDT 2017
$ ruby -Ilib bin/fpm --verbose -s gem -t deb json
$ dpkg-deb -x *.deb tmp
$ zmore tmp/usr/share/doc/rubygem-json/changelog.gz 
rubygem-json (2.1.0) whatever; urgency=medium

  * Package created with FPM.

 -- Dan Kegel <dank@kegel.com>  Wed, 31 Dec 1969 16:02:03 -0800
```